third_party/blink/renderer/core/frame/csp/source_list_directive.cc

```diff
@@ -106,7 +157,11 @@ bool CSPSourceListIsNone(
          !source_list.allow_unsafe_hashes && !source_list.allow_eval &&
          !source_list.allow_wasm_eval && !source_list.allow_wasm_unsafe_eval &&
          !source_list.allow_dynamic && !source_list.nonces.size() &&
+#if BUILDFLAG(IS_COBALT)
+         !source_list.hashes.size() && !source_list.cobalt_insecure_local_network;
```

Comment:
Nit: This looks a little odd with source_list.hashes repeated with #else clause below - does ours need to come last ?

Reply:
Nevermind it looks pretty consistent with the stuff below actually.